modules/arm_plugin/src/opset/utils.hpp

```diff
@@ -10,6 +10,12 @@
 #include "arm_compute/core/QuantizationInfo.h"
 #include "arm_compute/core/Types.h"
 
+#ifndef OV_NEW_API
```

Comment:
Should this define be moved to OpenVINO? Or there is no such issues in other plugins and modules?

Reply:
@ilyachur looks like we don't have good solution for template likes Variant?

Reply:
Yes it is right, we don't have a good backward compatible solution

Reply:
Looks like different plugins and modules can use different ways to resolve the issue.